src/exoticatechnologies/util/Extensions.kt

```diff
@@ -673,14 +673,23 @@ fun List<String>.containsIgnoreCase(string: String?): Boolean {
 fun getChildModuleVariantList(fleetMemberAPI: FleetMemberAPI): List<ShipVariantAPI> {
     val retVal = mutableListOf<ShipVariantAPI>()
     if (fleetMemberAPI.variant.moduleSlots == null || fleetMemberAPI.variant.moduleSlots.isEmpty()) return emptyList()
-    val moduleSlotList = fleetMemberAPI.variant.moduleSlots
+    var moduleSlotList = fleetMemberAPI.variant.moduleSlots.asSortedStringList()
```

Comment:
could be `val` again, since we're not doing anything here - it's already changed, sorted, then changed back to immutable list in the extension method.

Reply:
done.

Reply:
resolving.